src/Icon/index.jsx

```diff
@@ -24,9 +24,18 @@ function Icon(props) {
 }
 
 Icon.propTypes = {
+  // eslint-disable-next-line max-len
```

Comment:
Maybe we add ignoreComments to this rule for this project?
https://eslint.org/docs/rules/max-len

Reply:
Adding that causes some linting errors elsewhere, that aren't caused by comments for some reason. I don't want to break the `.eslintrc` on this PR, but I can come back and fix this.